packages/lib/getFeatures/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## unpublished
 
 - Breaking: `getWfsFeatures` now throws errors if required parameters on the wfs configuration are missing instead of only printing error messages on the console.
+- Feature: addressSearch.searchMethodsObject.queryParameters (type:wfs) now supports custom attributes for the like filter operator.
```

Comment:
This package can be used independently to `@polar/plugin-address-search` and thus the changelog entry should reflect the changes to this package. Something like

```md
- Feature: `getWfsFeatures` now offers the possibility to adjust the `wildCard`, `singleChar` and `escapeChar` properties of the `PropertyIsLike`-filter.
```

should be fine. Feel free to alter the text as you please though.

Reply:
I altered the text. It is not only possible to adjust the `wildCard`, `singleChar` and `escapeChar` attributes, it is possible to add any attributes to the `PropertyIsLike` operator.

Reply:
📞 ✔️